### PR TITLE
 Locked attributes (#116)

### PR DESCRIPTION
**Pull request by allan-ayamah**
Added config option "lockedAttrs"  which doesn't allow the user to delete attributes that are listed in this array.

Example: 
config: {
          label: 'Email',
          disabledAttrs: ['type'],
          lockedAttrs: ['required', 'className']
}

### CODE REVIEW

File: src/js/components/fields/edit-panel-item.js [hunk 1]
```diff
@@ -374,6 +374,14 @@ export default class EditPanelItem {
   }
 
   get itemControls() {
+    if (this.isLocked) {
```
Comment (kevinchappell) [inline src/js/components/fields/edit-panel-item.js:377]: After this PR I'm going to make a change so we can just return `null` here

**Comment by kevinchappell** [comment]
Great work!

**Comment by matthew798** [comment]
Can you disable attributes in the FormeoEditor configuration?

**Pull request merged**